src/ride/controllers/search_ride_controller.py

```diff
@@ -60,9 +61,10 @@ def process(self):
             ):
                 paths.append(path)
 
+        tz = zoneinfo.ZoneInfo("America/New_York")
```

Comment:
can we not use the `TIME_ZONE` variable here?

Reply:
u right